contributing/code_of_conduct/enforcement_team.rst

```diff
@@ -0,0 +1,66 @@
+Enforcement team
+================
+
+The role of the enforcement team is to make sure the :doc:`Code of Conduct </contributing/code_of_conduct/index>`
+is respected. If you think there is or has been a violation to the code of conduct
+please read the :doc:`reporting guidelines </contributing/code_of_conduct/reporting_guidelines>`
+and contact any person in the enforcement team.
+
+Members
+-------
+
+Lukas Kahwe Smith
```

Comment:
I don't like the idea of adding photos, biographies and personal links of the enforcement team people. We never do that in Symfony. We don't even show this for the Core Team, which is more important than this team. Listing the names and links to their SL Connect websites could be enough.

Reply:
This page will work as a contact page. So contact information is the most important thing here. 

I do not prefer a shared email. I *believe* some people may be more comfortable contacting a specific person. 

-----------

Photos and biographics etc is just to build trust and add personalities. I do not feel as strongly to this data as I do with the contact info.

Reply:
yeah and it will be up to each enforcement team member to decide what they want to publish about themselves to build trust.

Reply:
I still disagree. I thought there was going to be a central person-less email (`coc@symfony.com`) or form (`symfony.com/coc`) to submit CoC violations. What's this thing about personal emails and contacting people individually?

Reply:
because lets say someone wants to report someone that is either on or very closely related to someone on the enforcement team. or someone simply has had very bad experiences and first needs to build more trust.

Reply:
Why not both?

Reply:
I've added a common email and I've removed the images.